elifetools/tests/fixtures/test_xmlio/test_reparsed_tag_01_expected.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" ?><root xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:jats="http://www.ncbi.nlm.nih.gov/JATS1" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Test &amp; <i class="a">a</i> <mml:math><mml:mi>K</mml:mi></mml:math>.</root>
```

Comment:
using some new lines here could make it more readable?

Reply:
In a `prod` situation, I would expect we'd use XML with no additional whitespace for best results. Considering this is just a test case, I can switch it over to invoke `reparsed.toprettyxml()` instead of just `reparsed.toxml()` and I'll change the text fixture to reflect the pretty XML.